submission/persistence-sql/src/test/kotlin/ac/uk/ebi/biostd/persistence/repositories/data/SubmissionRepositoryTest.kt

```diff
@@ -0,0 +1,65 @@
+package ac.uk.ebi.biostd.persistence.repositories.data
+
+import ac.uk.ebi.biostd.persistence.mapping.extended.to.ToExtSubmissionMapper
+import ac.uk.ebi.biostd.persistence.model.DbSubmission
+import ac.uk.ebi.biostd.persistence.repositories.SectionDataRepository
+import ac.uk.ebi.biostd.persistence.repositories.SubmissionDataRepository
+import ac.uk.ebi.biostd.persistence.repositories.SubmissionRequestDataRepository
+import ac.uk.ebi.biostd.persistence.repositories.SubmissionStatsDataRepository
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.mockkStatic
+import io.mockk.unmockkStatic
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import uk.ac.ebi.extended.serialization.service.ExtSerializationService
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+@ExtendWith(MockKExtension::class)
+class SubmissionRepositoryTest(
```

Comment:
Should we also check in the integrations tests that the date is properly modified?

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/security/DeletePermissionTest.kt#L66

Reply:
not sure to be honest, more test is usually never bad but actually such going such low level IT can increase its complexity and make it harder to identify code business conditions.  Modification date is not even publised to users.